packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -355,7 +355,8 @@ def _get_font(self, prop):
     def get_text_width_height_descent(self, s, prop, ismath):
         if ismath:
             image, d = self.mathtext_parser.parse(s, self.dpi, prop)
-            w, h = image.get_width(), image.get_height()
+            image_arr = np.asarray(image)
+            h, w = image_arr.shape
```

Comment:
See <https://matplotlib.org/3.3.0/api/prev_api_changes/api_changes_3.2.0/deprecations.html#fonts>. This should have been producing a FutureWarning in 3.3 already, are those not getting displayed? Ideally warnings should be errors in tests.

Reply:
I wonder if this is showing up in https://github.com/pyodide/pyodide/pull/2194, actually…